app/policies/rent_policy.rb

```diff
@@ -0,0 +1,19 @@
+class RentPolicy < ApplicationPolicy
+  class Scope
+    attr_reader :user, :scope
```

Comment:
This attributes was inherited from parent class

Reply:
No it isn't because the parent class is Object

Reply:
I didn't know how to explain myself, but Marian understood me :heart_eyes: 
```rb
class Scope < ApplicationPolicy::Scope
```